src/endpoint.cpp

```diff
@@ -709,22 +763,41 @@ bool UartEndpoint::setup(UartEndpointConfig conf)
     for (auto msg_id : conf.allow_msg_id_out) {
         this->filter_add_allowed_out_msg_id(msg_id);
     }
+    for (auto msg_id : conf.block_msg_id_out) {
+        this->filter_add_blocked_out_msg_id(msg_id);
+    }
     for (auto src_comp : conf.allow_src_comp_out) {
         this->filter_add_allowed_out_src_comp(src_comp);
     }
+    for (auto src_comp : conf.block_src_comp_out) {
+        this->filter_add_blocked_out_src_comp(src_comp);
+    }
     for (auto src_sys : conf.allow_src_sys_out) {
         this->filter_add_allowed_out_src_sys(src_sys);
     }
+    for (auto src_sys : conf.block_src_sys_out) {
+        this->filter_add_blocked_out_src_sys(src_sys);
+    }
+
```

Comment:
Again: Why the extra newline? There already was one to separate the two "blocks" of for each loops.

Reply:
clang-format has the same question by the way